assets/controllers/invoice-sync-worklogs_controller.js

```diff
@@ -0,0 +1,56 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+    updateUrl;
+    connect() {
+        this.updateUrl = this.element.dataset.updateUrl;
+    }
+
+    syncWorklogs(e) {
```

Comment:
Cleanup implementation so html is in the twig component and this controller just changes state of the html